tutorials/graphics/tmathtext.C

```diff
@@ -29,5 +29,5 @@
    c1->Print("c1.ps");
    c1->Print("c1.eps");
 
-   return c1;
+   return 0; // success
```

Comment:
Returning `c1` means that `c1` will get included in the doc of this tutorial. So this should not be changed.